jetstream/options.go

```diff
@@ -402,6 +402,14 @@ func MetaOnly() WatchOpt {
 	})
 }
 
+// ResumeFromSeq instructs the key watcher to resume from a specific sequence number.
+func ResumeFromSeq(revision uint64) WatchOpt {
```

Comment:
As this is KV abstraction over Stream, we should probably use KV naming convetions.

This should become `ResumeFromRevision`.

Reply:
Related Sequence names have been updated to Revision. 

Please take a look, thanks!